types/html2pdf.js/index.d.ts

```diff
@@ -0,0 +1,348 @@
+        run<TResult1 = any, TResult2 = never>(
+            onfulfilled?: ((value: any) => TResult1 | PromiseLike<TResult1>) | null,
+            onrejected?: ((reason: any) => TResult2 | PromiseLike<TResult2>) | null,
+        ): Worker;
+    }
+
+    /**
+     * Worker constructor
+     */
+    const Worker: {
+        new(options?: Options): Worker;
+    };
+
+    /**
+     * Options for html2pdf configuration
+     */
+    interface Options {
+        /**
+         * The default filename of the exported PDF
+         * @default 'file.pdf'
+         */
+        filename?: string | undefined;
+
+        /**
+         * PDF margin (in jsPDF units). Can be a single number, [vMargin, hMargin], or [top, left, bottom, right]
+         * @default [0,0,0,0]
+         */
+        margin?: number | number[] | undefined;
+
+        /**
+         * The image type and quality used to generate the PDF
+         * @default { type: 'jpeg', quality: 0.95 }
+         */
+        image?: ImageOptions;
+
+        /**
+         * If enabled, PDF hyperlinks are automatically added ontop of all anchor tags
+         * @default true
+         */
+        enableLinks?: boolean | undefined;
+
+        /**
+         * Controls the pagebreak behaviour on the page
+         * @default { mode: ['css', 'legacy'] }
+         */
+        pagebreak?: PagebreakOptions | undefined;
+
+        /**
+         * Configuration options sent directly to html2canvas
+         */
+        html2canvas?: any;
+
+        /**
+         * Configuration options sent directly to jsPDF
+         */
+        jsPDF?: any;
+    }
+
+    /**
+     * Image options for PDF generation
+     */
+    interface ImageOptions {
+        /**
+         * The image type. HTMLCanvasElement only supports 'png', 'jpeg', and 'webp' (on Chrome)
+         * @default 'jpeg'
+         */
+        type?: ImageType | undefined;
+
+        /**
+         * The image quality, from 0 to 1. This setting is only used for jpeg/webp (not png)
+         * @default 0.95
+         */
+        quality?: number | undefined;
+    }
+
+    /**
+     * Pagebreak options for controlling page breaks
+     */
+    interface PagebreakOptions {
+        /**
+         * The mode(s) on which to automatically add page-breaks
+         * @default ['css', 'legacy']
+         */
+        mode?: PagebreakMode | Array<PagebreakMode> | undefined;
+
+        /**
+         * CSS selectors for which to add page-breaks before each element
+         * @default []
+         */
+        before?: string | string[] | undefined;
+
+        /**
+         * CSS selectors for which to add page-breaks after each element
+         * @default []
+         */
+        after?: string | string[] | undefined;
+
+        /**
+         * CSS selectors for which to avoid page-breaks
+         * @default []
+         */
+        avoid?: string | string[] | undefined;
+    }
+}
+
+// Global namespace for browser usage
+declare global {
```

Comment:
for browsers, you need `export as namespace html2pdf` instead of `declare global`. The namespace here looks like a duplicate of the one above and won't be needed if you have `export as namespace`.